### PR TITLE
Use gRPC health check for runme server

### PR DESCRIPTION
**Pull request by mxsdev**
Detect whether gRPC server exists using `grpc/grpc` healthcheck rather than using the parser client directly. This is more in line with gRPC standard practice and gives us more options down the line.

### CODE REVIEW

File: src/extension/server/runmeServer.ts [hunk 1]
```diff
@@ -5,12 +5,15 @@ import path from 'node:path'
 import { ChannelCredentials } from '@grpc/grpc-js'
 import { GrpcTransport } from '@protobuf-ts/grpc-transport'
 import { Disposable, Uri, EventEmitter } from 'vscode'
+import {
+  HealthCheckRequest,
+  HealthCheckResponse_ServingStatus
+} from '@buf/grpc_grpc.community_timostamm-protobuf-ts/grpc/health/v1/health_pb'
```
Comment (sourishkrout) [inline src/extension/server/runmeServer.ts:11]: More of a nit, don't love bleeding the full module name outside the "grpc" namespace. I'd suggest to import and export them just like we have with other generated types.

**Review by sourishkrout — CHANGES_REQUESTED** [review]
See namespacing comment. Granted it passed functional testing on both Windows and Mac, good to go 👍 .

File: src/extension/server/runmeServer.ts (hunk 1 above)
Comment (mxsdev) [reply src/extension/server/runmeServer.ts]: See 4432482

**Review by sourishkrout — APPROVED** [review]
👍 LGTM

**Pull request merged**